passes/cmds/select.cc

```diff
@@ -1113,6 +1113,9 @@ struct SelectPass : public Pass {
 		log("    <obj_pattern>\n");
 		log("        select the specified object(s) from the current module\n");
 		log("\n");
+		log("Prefix the following patterns with '=' if the pattern should match black-/\n");
```

Comment:
It says "the _following_ patterns" but actually that's in the middle of the section describing patterns.

Reply:
I moved it from being in between describing mod_pattern and obj_pattern to before both, or it could after both.
Would this be clearer?
```suggestion
		log("Prefix <[mod|obj]_pattern> with '=' if the pattern should match black-/\n");
```

Reply:
Why not just "prefix the pattern"? I just don't like the use of "following" at a position that does not precede the section on patterns but is in the middle of it.  (moving it up is fine and definitely an improvement but it's still between the paragraphs that outline the general format and the paragraphs the explain the details.)

The issue I have with "Prefix <[mod|obj]_pattern>" is that in `<mod_pattern>/<obj_pattern>` you can put the prefix before  `<mod_pattern>`, but not `<obj_pattern>`, because it's a prefix to the entire pattern, not one of the two components.